<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillsWidget</class>
 <widget class="QWidget" name="SkillsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Configure Skill</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="3">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QSpinBox" name="SB_DISTANCECAST">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="PB_DEFAULT">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Default</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QComboBox" name="CB_TARGET">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Own character</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Monster</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Own pet</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Own partner</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="PB_SAVE">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Priority :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Distance max of cast :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="RB_CASTAVAILABLE">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Cast when available</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="LE_SKILLNAME">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Target :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="RB_CASTSECONDS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Cast every</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QSpinBox" name="SB_PRIORITY">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QSpinBox" name="SB_CASTSECOND">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Skill name :</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QLabel" name="L_SAVESTATUS">
        <property name="maximumSize">
         <size>
          <width>123123</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Skills</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QPushButton" name="PB_DELETE">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Available skills :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Active skills :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="PB_CLEAR">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QListWidget" name="LW_ACTIVESKILLS"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QListWidget" name="LW_SKILLS"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
